### PR TITLE
Removing placeholder saas offerings

### LINKED ISSUE #1499: SaaS offerings in new landing page should be empty by default
We are currently defaulting to a set of placeholder/example SaaS offerings. We need to remove this placeholders.

### PR DESCRIPTION
**Pull request by rhamilto**
Fixes #1499

### CODE REVIEW

**Comment by rhamilto** [comment]
[test]

**Comment by spadgett** [comment]
Lets leave an example commented out so someone looking at this file knows the format

**Comment by rhamilto** [comment]
> Lets leave an example commented out so someone looking at this file knows the format

I wondered the same thing.  I've added an example.

File: app/scripts/constants.js [hunk 1]
```diff
@@ -441,9 +441,8 @@ angular.extend(window.OPENSHIFT_CONSTANTS, {
...
-    {id: 2, title:  'Mobile Application', icon: 'fa fa-mobile',  url: 'https://www.redhat.com/en/technologies/mobile', description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt.'},
-    {id: 3, title:  'Integration Application', icon: 'fa fa-plug',  url: 'https://www.redhat.com/en/technologies/cloud-computing', description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt.'},
-    {id: 4, title:  'Business Process Application', icon: 'fa fa-cubes',  url: 'https://www.redhat.com/en/technologies/management', description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt.'},
+    // empty by default
+    // EXAMPLE
+    // {id: 1, title:  'Microservices Application', icon: 'fa fa-cubes',  url: 'https://www.example.com/', description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt.'},
```
Comment (spadgett) [inline app/scripts/constants.js]: Nit: remove the trailing comma since it's invalid JSON if you just uncomment this line

**Comment by spadgett** [comment]
@jwforres OK to go ahead and make this change? I know we're waiting on https://github.com/openshift/origin-web-catalog/issues/198, but we'll need to remove these either way.

**Comment by jwforres** [comment]
yes go ahead, it does mean QE will need to configure SaaS offerings to test
any updates to the SaaS section so we'll need to be clear on any cards for
that

**Comment by spadgett** [comment]
[merge]

**Pull request merged**